### ISSUE TITLE
每隔若干帧，调用yolo来更新跟踪点

### ISSUE THREAD

**Issue by LLsetnow** [LLsetnow/ComfyMask#2]
predictor.reset_state(inference_state) # 重置跟踪 用于重置跟踪点
    # 从frame_idx帧 增加跟踪点
    _, _, out_mask_logits = predictor.add_new_points_or_box(
        inference_state=inference_state,
        frame_idx=DrawPointFrame,
        obj_id=1,
        points=points,
        labels=labels,
    )
用于为第frame_idx帧 添加跟踪点
